test/sql/create_pg/create_property_graph.test

```diff
@@ -51,6 +51,13 @@ EDGE TABLES (
             PROPERTIES ( createDate ) LABEL Knows
     )
 
+statement ok
+-SELECT count(id)
+FROM
+  GRAPH_TABLE (PG 
```

Comment:
The property graph created at L43 is of lower case.

Reply:
Just to double check, could you change the test to verify it has the right result? Instead of only checking that the query doesn't crash :)

Reply:
Updated! PTAL, thanks!

Reply:
Thanks! LGTM